src/service/v2_game_play_log.go

```diff
@@ -0,0 +1,18 @@
+package service
+
+import (
+	"context"
+	"time"
+
+	"github.com/traPtitech/trap-collection-server/src/domain"
+	"github.com/traPtitech/trap-collection-server/src/domain/values"
+)
+
+//go:generate go tool mockgen -source=$GOFILE -destination=mock/${GOFILE} -package=mock
```

Comment:
`-typed`を付けると生成されるコードの`Return`とかに型がついて嬉しいです
```suggestion
//go:generate go tool mockgen -source=$GOFILE -destination=mock/${GOFILE} -package=mock -typed
```